duckduckgo/iOS
# update application lock as per zeplin designs

Reviewer: Caine
Asana:
CC:

**Description**:

* Removes icon from settings and updates copy
* Removes lock icon from Dax on authentication screen
* Use new generic unlock icon when prompting user to tap to authenticate

**Steps to test this PR**:

1. Go to settings - observe no icon on application lock setting.  Check copy is correct.
1. Turn on Application lock
1. Leave app
1. Launch app and fail authentication - observe prompt.  Check copy and icon are situated correctly.
1. Tap to authenticate
1. Authenticate and access app
